lib/Autocomplete.js

```diff
@@ -55,7 +56,7 @@ let Autocomplete = React.createClass({
   },
 
   componentWillMount () {
-    this.id = lodash.uniqueId('autocomplete-'); 
+    this.id = this.props.id || lodash.uniqueId('autocomplete-'); 
```

Comment:
After #89 is merged you'll still have the ability to assign an id to the input via `inputProps` (which you should to make labels focus the corresponding input).